### PR TITLE
Including sass-rails image-url-helper for the usage of the asset pipeline.

### LINKED ISSUE #272: some background images not shown in production
If you look at the error log, you can see som 404 errors on some images:

"GET /assets/layout/bg.png HTTP/1.1" 404 1564 "https://portus.suse.example.com/assets/application-10b79a6ebcc701ac07c9ebc278de7bcc.css" "Mozilla/5.0 (X11; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0

"GET /assets/layout/bg-login.png HTTP/1.1" 404 1564 "https://portus.suse.example.com/assets/application-10b79a6ebcc701ac0$c9ebc278de7bcc.css" "Mozilla/5.0 (X11; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0"

The problem is that it should be looking for the compiled assets, like 

GET /assets/layout/MD5SUMbg.png

I think the affected pages are

app/assets/stylesheets/layout.scss
app/assets/stylesheets/variables.scss

where we should be using the image_tag as in 

<%= image_tag "layout/bg.png" %>

reference:
http://guides.rubyonrails.org/asset_pipeline.html

### PR DESCRIPTION
**Pull request by jloehel**
Fixes #272

Signed-off-by: Jürgen Löhel <jloehel@suse.com>

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -42,11 +42,11 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  # config.force_ssl = false
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  # config.log_level = :debug
+  config.log_level = :info
```
Comment (flavio) [inline config/environments/production.rb]: This is not related with the PR, plus it has just been fixed on master. Can you rebase against master and update your PR?

Comment (jloehel) [inline config/environments/production.rb]: Of course, I've just updated the PR.

**Comment by flavio** [comment]
LGTM
